FALSEHOODS.md

```diff
@@ -0,0 +1,79 @@
+
+    Old phone numbers are recycled and get reassigned to other people.
+
+1.  **Each country calling code corresponds to exactly one country**
+
+    The USA, Canada, and several Caribbean islands share the country calling code +1. Russia and Kazakhstan share +7. These are not the only examples!
+
+1.  **Each country has only one country calling code**
+
+    As of this present moment (in Mar. 2016), phones in the disputed territory and partially recognised state of Kosovo may be reached by dialing the country calling code for Serbia (+381), Slovenia (+386), or Monaco (+377), depending on where and when one obtained the number.
+
+    *Tip:* Use the phone widget to encourage users to enter their phone number in an international format such that we can understand it.
+
+1.  **A phone number is dialable from anywhere**
+
+    Some numbers can only be dialed within the country. Some can only be dialled from within a subset of countries, such as the international 00800 numbers. Some may be dialable only if the caller is a subscriber to a particular telecom company.
+
+1.  **There are only two ways to dial a phone number: domestically and from overseas**
+
+    Some numbers may need different prefixes depending on: the carrier you are using; what device you are dialling from/to; whether you are inside or outside a particular geographical region.
+
+    Examples:
+    *   In Brazil, to dial numbers internally but across a certain geographical boundary, a carrier code must be explicitly dialed to say which carrier you will use to pay for the call.
+    *   In Nepal, the leading zero in national format is omitted depending on whether the originating phone is mobile or fixed-line.
+    *   In New Zealand, you need to dial the area-code (e.g. 03) even if the number is within the same area-code region as you are, unless it is "close" (something approximating city/district boundaries), in which case it shouldn’t be dialled.
+
+    *Tip:* Use formatForMobileDialling to get the number a user should actually dial on their mobile phone.
+
+1.  **To make a number dialable, you only need to change the prefix**
+
+    In Argentina, to dial a mobile number domestically, the digits "15" need to be inserted *after* the area code but *before* the local number, and the "9" after the country code (54) needs to be removed. This transforms +54 9 2982 123456 into 02982 15 123456.
+
+1.  **No prefix of a valid phone number can be a valid phone number**
+
+    In some countries, it's possible to connect to a different endpoint by dialing more digits after a number. So "12345678" may not reach the same person as dialing "123456".
+
+1.  **An invalid number will not reach an endpoint**
+
+    In some countries, or on some phones, extra digits are thrown away. Hence, 1-800-MICROSOFT is an invalid number - but it still connects to Microsoft, since any later digits are ignored. Numbers such as "911" can be reached by dialling "911 123" in some countries: but not in others.
```

Comment:
I'm not actually sure that vanity numbers are understood everywhere. I hadn't seem them til I lived in the States. I wonder if some readers won't understand 1-800-MICROSOFT. Maybe have a footer explaning that this would dial 1-800-648767638?